openapi/openapi.json

```diff
@@ -368,209 +550,285 @@
         "summary" : "Aggregate",
         "description" : "Aggregate the elements in the collection(s), given the filters and the aggregation parameters",
         "operationId" : "aggregatePost",
-        "consumes" : [ "application/json;charset=utf-8" ],
-        "produces" : [ "application/json;charset=utf-8" ],
         "parameters" : [ {
           "name" : "collection",
           "in" : "path",
           "description" : "collection",
           "required" : true,
-          "type" : "string"
-        }, {
-          "in" : "body",
-          "name" : "body",
-          "required" : false,
           "schema" : {
-            "$ref" : "#/definitions/AggregationsRequest"
+            "type" : "string"
           }
         }, {
           "name" : "pretty",
           "in" : "query",
           "description" : "Pretty print",
-          "required" : false,
-          "type" : "boolean",
-          "default" : false
+          "schema" : {
+            "type" : "boolean",
+            "default" : false
+          }
         }, {
           "name" : "max-age-cache",
           "in" : "query",
           "description" : "max-age-cache",
-          "required" : false,
-          "type" : "integer",
-          "format" : "int32"
+          "schema" : {
+            "type" : "integer",
+            "format" : "int32"
+          }
         } ],
+        "requestBody" : {
+          "content" : {
+            "application/json;charset=utf-8" : {
+              "schema" : {
+                "$ref" : "#/components/schemas/AggregationsRequest"
+              }
+            }
+          }
+        },
         "responses" : {
           "200" : {
             "description" : "Successful operation",
-            "schema" : {
-              "$ref" : "#/definitions/AggregationResponse"
+            "content" : {
+              "application/json;charset=utf-8" : {
+                "schema" : {
+                  "type" : "array",
```

Comment:
will be fixed by removing the @array annotation

Reply:
ok